robyn/robyn.pyi

```diff
@@ -293,6 +295,9 @@ class Response:
     response_type: Optional[str] = None
     file_path: Optional[str] = None
 
+    def set_cookie(self, key, value) -> None:
```

Comment:
@VishnuSanal , please add a docstring in the public functions. And also add typing here